LvivCompany.Bookstore/LvivCompany.Bookstore.Web/wwwroot/css/site.css

```diff
@@ -41,3 +41,14 @@ input[type="checkbox"].input-validation-error {
     display: none;
 }
 
+.img-grid {
```

Comment:
Remove one of the below classes. Also for future, try to use CSS selectors if possible, otherwise create new classes.